app/assets/stylesheets/puppetclasses.scss

```diff
@@ -32,8 +32,12 @@
   .matcher .matcher-group {
     display: block;
   }
-  
+
   .matcher_value {
     width: 60%;
   }
 }
+
+.selected_puppetclass.unavailable a {
+  color: #d1d1d1;
```

Comment:
can't we simply access it via js/css directly? would make it both readable and if pf colors change we will have less of an headache :)